utils/docker/setup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # Start MySQL
-/usr/bin/mysqld_safe & 
+service mysql start
```

Comment:
This is interesting.. not.. 

Should we not be using systemctl?  I know the backwards compatability stuff pretty much works... but not always..